packages/runtime-common/matrix-constants.ts

```diff
@@ -12,3 +12,12 @@ export const APP_BOXEL_REALM_SERVER_EVENT_MSGTYPE =
 export const APP_BOXEL_ROOM_SKILLS_EVENT_TYPE = 'app.boxel.room.skills';
 export const APP_BOXEL_REALMS_EVENT_TYPE = 'app.boxel.realms';
 export const LEGACY_APP_BOXEL_REALMS_EVENT_TYPE = 'com.cardstack.boxel.realms';
+export const APP_BOXEL_ACTIVE_LLM = 'app.boxel.active-llm';
+
+export const DEFAULT_LLM = 'openai/gpt-4o';
+export const DEFAULT_LLM_LIST = [
+  'anthropic/claude-3.5-sonnet',
+  'google/gemini-pro-1.5',
+  'openai/gpt-4o',
+  'openai/gpt-4o-mini',
+];
```

Comment:
Not sure if this is the right place to put `DEFAULT_LLM` and `DEFAULT_LLM_LIST` constants but I put them here to make it available for host and ai-bot.